### ISSUE TITLE
Persistent storage

### ISSUE THREAD

**Issue by PeeHaa** [PeeHaa/mailgrab#13]
Optional persistent storage of mails (maybe redis storage?)

**Comment by shadowhand** [comment]
SQLite would be ideal, because then the Phar is completely self contained. Maybe also automatically purge emails >30d old.

**Comment by PeeHaa** [comment]
Yeah that was my first thought too (re SQLite). Need to see if/how I can make it async compatible.
